webpack.config.js

```diff
@@ -139,8 +139,9 @@ const config = [
         module: mod,
     },
     // Modules
-    lorisModule('media', ['CandidateMediaWidget', 'mediaIndex']),
     lorisModule('issue_tracker', ['issueTrackerIndex', 'index']),
+    lorisModule('media', ['CandidateMediaWidget', 'mediaIndex']),
+    lorisModule('issue_tracker', ['issueTrackerIndex', 'index', 'CandidateIssuesWidget']),
```

Comment:
Why are there two 'issue_tracker's?

Reply:
poorly resolved rebase conflict.. should be fixed.